src/engine/enginebuffer.cpp

```diff
@@ -1196,15 +1196,24 @@ void EngineBuffer::postProcess(const int iBufferSize) {
 }
 
 void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
-    VERIFY_OR_DEBUG_ASSERT(m_trackSampleRateOld && m_tempo_ratio_old) {
-        // no track loaded, function not called in this case
-        return;
+    if (!m_trackSampleRateOld) {
+       // this happens if Deck Passthrough is active but no track is loaded
+       // skip indicator updates.
+       return;
     }
 
     // Increase samplesCalculated by the buffer size
     m_iSamplesSinceLastIndicatorUpdate += iBufferSize;
 
     const double fFractionalPlaypos = fractionalPlayposFromAbsolute(m_filepos_play);
+    
+    double ratio = m_tempo_ratio_old;  
+    if (ratio == 0.0) {
+	// In case the track is slowed done to zero we will have INF relaining seconds
+        // We jump back to a rate of 1.0 to show a useful time. 
```

Comment:
inconsistent tabbing in this block

Reply:
fix typo: "In case the track is slowed done to zero we will have INF remaining seconds.
        // We jump back to a rate of 1.0 to show a useful time."

Reply:
@daschuer Ping

Reply:
Also fixed